i18n.py

```diff
@@ -16,25 +16,26 @@
 
 # Running params
 params = {"recursive": False,
```

Comment:
Why is this file included at all? Feel free to remove it and link to https://github.com/minetest-tools/update_translations instead.